<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiSearchWidget</class>
 <widget class="QWidget" name="WifiSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
       <item>
        <widget class="CustomLabel" name="filterNameLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="CustomPushButton" name="applyPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CustomPushButton" name="yesPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CustomPushButton" name="noPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>CustomPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">CustomLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
